protocol_tests/__init__.py

```diff
@@ -222,3 +225,26 @@ def temporary_channel(
     """
     async for _event, *_data in generator:
         pass
+
+
+class BackChannel:
+    def __init__(self, endpoint):
+        self.endpoint = endpoint
+        self.headers = {'content-type': 'application/json'}
+
+
+    class Action(Enum):
+        START_CONNECTION_AS_INVITER = "start_connection_as_inviter"
+        START_CONNECTION_AS_INVITEE = "start_connection_as_invitee"
+        GET_CONNECTION_STATE = "get_connection_state"
```

Comment:
A method is not yet defined on the Connection Backchannel ABC for getting connection state but we should be able to add this without any trouble.